serializer.rst

```diff
@@ -68,6 +68,13 @@ As well as the following normalizers:
 * :class:`Symfony\\Component\\Serializer\\Normalizer\\ArrayDenormalizer` to
   denormalize arrays of objects using a format like `MyObject[]` (note the `[]` suffix)
 
+.. note::
+
+    If you installed the serializer directly through `composer require symfony/serializer`,
+    the :class:`Symfony\\Component\\Serializer\\Normalizer\\ObjectNormalizer` will
+    not be available without `composer require symfony/property-access`.
```

Comment:
We don't love notes like these :/. The problem is that each note has a "cost" - it adds extra stuff that's not relevant for most users to worry about. I'd vote to fix the install command above, and if you're gone on your own track, you're on your own :)

Reply:
Alright, no note it is. ^_^